<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectionCouleurs</class>
 <widget class="QWidget" name="DetectionCouleurs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>DETECTION COULEURS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="layoutCouleurs">
     <item>
      <widget class="CouleursChoix" name="w_C1" native="true"/>
     </item>
     <item>
      <widget class="CouleursChoix" name="w_C2" native="true"/>
     </item>
     <item>
      <widget class="CouleursChoix" name="w_C3" native="true"/>
     </item>
     <item>
      <widget class="CouleursChoix" name="w_C4" native="true"/>
     </item>
     <item>
      <widget class="CouleursChoix" name="w_C5" native="true"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nombre Couleurs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CouleursChoix</class>
   <extends>QWidget</extends>
   <header location="global">couleurschoix.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
